ndk/src/events/content-tagger.ts

```diff
@@ -163,7 +163,7 @@ export async function generateContentTags(
     await Promise.all(promises);
 
     content = content.replace(hashtagRegex, (tag, word) => {
-        const t: NDKTag = ["t", word];
+        const t: NDKTag = ["t", word.slice(1)];
```

Comment:
Fixes an issue with content tagging hashtags where the `#` char was being included in the tag.